File: src/php/Rpc/Services/UserCenter/UserCenter.php
<?php
/**
 * Created by PhpStorm.
 * User: zhangyubo
 * Date: 2019/4/25
 * Time: 12:00
 */
namespace Cecd\Sdk\Rpc\Services\UserCenter;
use Cecd\Sdk\Rpc\Services\RpcModuleIf;
use Cecd\Sdk\Rpc\Services\RpcModuleTrait;

/**
 * Class UserCenter
 * @package Cecd\Sdk\Rpc\Services\UserCenter
 * @property \Cecd\Sdk\Rpc\Services\UserCenter\Models\MemberModel MemberModel
 * @property \Cecd\Sdk\Rpc\Services\UserCenter\Libraries\MemberLib MemberLib
 */
class UserCenter implements RpcModuleIf
{
    use RpcModuleTrait;

    private $serviceId = 1005;

    private $config_key = 'rpc.user-center';
}

File: src/java/src/main/java/com/cecd/sdk/examples/usercenter/UserCenter.java
package com.cecd.sdk.examples.usercenter;

import com.cecd.sdk.examples.RpcClientIntercepterTest;
import com.cecd.sdk.rpc.RpcFactory;
import com.cecd.sdk.rpc.RpcModuleAbstract;
import com.cecd.sdk.rpc.RpcModuleIf;

public class UserCenter extends RpcModuleAbstract {


    public UserCenter() {
        //初始化后放入工厂类中
        this.setInterceptor(RpcClientIntercepterTest.class);
        this.setServiceId(1005);
        this.setServiceName("user-center");
        this.setLang("php");
        RpcFactory.addService(this);
    }


}
